coding-guides/cleverbot-integration.md

```diff
@@ -9,16 +9,16 @@ I've had this request since I started An Idiot's Guide, in fact it was one of th
 So to get started, let's grab the example from [getting started](../getting-started/getting-started-long-version.md) and shove it in a file.
 
 ```javascript
-const Discord = require("discord.js");
-const client = new Discord.Client({
-  intents: ["GUILDS", "GUILD_MESSAGES"]
+const { Client, Intents: { FLAGS: { GUILDS, GUILD_MESSAGES } } } = require("discord.js");
+const client = new Client({
+  intents: [GUILDS, GUILD_MESSAGES]
 });
```

Comment:
To keep things consistent with this line here: [`sqlite-based-points-system.md`](https://github.com/AnIdiotsGuide/discordjs-bot-guide/pull/170/files#diff-51c8636766f0af606d940c35b829646f47506cc2009cfd132821dc07ed254a67R69)

```suggestion
const { Client, Intents } = require("discord.js");

const client = new Client({
  intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES]
});
```

Reply:
Fixed :+1: